netbox/wireless/forms/model_forms.py

```diff
@@ -181,3 +181,8 @@ class Meta:
             'auth_type': 'Type',
             'auth_cipher': 'Cipher',
         }
+        error_messages = {
+            'length': {
+                'max_value': _('Maximum length is 32767 (any unit)')
+            }
+        }
```

Comment:
I see that this was copied from Cable, but I don't believe it's necessary anymore. (`length` and `_abs_length` are decimals fields.) It should be removed from Cable as well.